operations/itunestesters/README.md

```diff
@@ -13,7 +13,7 @@ The process to add a user as an internal tester is as follows:
 
  1. After logging in, you will be presented with the following screen, **select 'users and access'**
  <p align="center">
-     <img src="/internal/itunesConnect_homepage.png">
+     <img src="Internal/itunesConnect_homepage.png">
```

Comment:
How come the URL change?

Reply:
The folder had a capital, you can't rename folders online, and the repo isnt tracking folder name changes :(

Reply:
So technically you can, you just have to move the files to a new directory (delete the old one), commit that, then move them back and commit again.

Reply:
this has been done, change is outdated